src/renderer/components/app.tsx

```diff
@@ -75,42 +75,20 @@ export class App extends React.Component {
     whatInput.ask(); // Start to monitor user input device
   }
 
-  async componentDidMount() {
-    const cluster = getHostedCluster();
-    const promises: Promise<void>[] = [];
+  componentDidMount() {
+    disposeOnUnmount(this, [
+      kubeWatchApi.subscribeStores([podsStore, nodesStore, eventStore], {
+        preload: true,
+      }),
```

Comment:
default is preload=true

Reply:
yes, here is used only for better visibility of what's going on

Reply:
> Is there a reason why we also want it to be bold?

I just used common style for both logging outputs, so no special reason as well as why not?

Reply:
Given that, should we just require that `preload` is always specified?

Reply:
Yes.

Reply:
I meant, changing `src/renderer/api/kube-watch-api.ts:28` to be `preload: boolean` instead of `preload?: boolean`.

Reply:
Why? It should not be a required field / not critical imho.. I put explicit `preload: true` for better visibility, we can remove it all as well since it's default value.

Reply:
What I mean is, I accept your reasoning that they should be explicit (for visibility reasons) but then I think that we should always require that. Just a suggestion though.